src/flavors/defaultflavor/config.yml

```diff
@@ -338,6 +338,27 @@ place:
       dataset: demo
       icon_url: /static/css/images/markers/marker-observation.png
       value: demo
+      hide_category_button_on_selection: true
```

Comment:
nit: I think this should be called `hide_category_button_on_select`, to be consistent with the `onSelect` event in the web api.

But also see my earlier comment where I don't think we need this flag at all, since it seems like we can derive it from `multi_stage`.